docs/guides/nrfconnect_examples_configuration.md

```diff
@@ -142,3 +142,55 @@ Read the
 guide in the nRF Connect SDK's Zephyr documentation if you are interested in
 getting more advanced and detailed information about the configuration
 structure.
+
+<hr>
+
+## Configuring Matter in nRF Connect platform
+
+### Mandatory configuration
+
+To use the Matter protocol, you need to set the `CONFIG_CHIP` Kconfig option. Setting this option enables the Matter protocol stack and other associated Kconfig options, including `CONFIG_CHIP_ENABLE_DNSSD_SRP` that is required for the Matter device to be discoverable using DNS-SD.
+
+After that, make sure to set the `CONFIG_CHIP_PROJECT_CONFIG` Kconfig option and define the path to the configuration file that specifies Vendor ID, Product ID, and other project-specific Matter settings.
+
+<hr>
+
+### Optional configuration
+
+After enabling the Matter protocol and defining the path to the Matter configuration file, you can enable additional options in Kconfig.
+
+**Sleepy End Device support**
+
+You can enable the support for Thread Sleepy End Device in Matter by setting the following Kconfig options:
+- `CONFIG_OPENTHREAD_MTD`
+- `CONFIG_CHIP_ENABLE_SLEEPY_END_DEVICE_SUPPORT`
+
+**Commissioning with NFC support**
+
+You can configure the Matter protocol to use an NFC tag for commissioning, instead of using a QR code, which is the default configuration.
+
+To enable NFC for commissioning and share the onboarding payload in an NFC tag, set the `CONFIG_CHIP_NFC_COMMISSIONING` option.
+
+**Logging**
+
+You can enable logging for both the stack and Zephyr’s [Logging](https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/zephyr/reference/logging/index.html#logging-api) API by setting the `CONFIG_LOG` option.
+
+Zephyr allows you to configure log levels of different software modules independently. To change the log level configuration for the Matter module, set one of the available options:
+- `CONFIG_MATTER_LOG_LEVEL_ERR`
+- `CONFIG_MATTER_LOG_LEVEL_INFO`
+- `CONFIG_MATTER_LOG_LEVEL_DBG`
+
+**Shell**
+
+You can enable the Matter shell library using the `CONFIG_CHIP_LIB_SHELL` Kconfig option. This option lets you use the Matter specific shell commands.
+See [Using CLI in nRF Connect examples](nrfconnect_examples_cli.md) for the list of available Matter shell commands.
+
+**Matter device identification**
+
+Matter has many mandatory and optional ways to identify a specific device. These can be used for various purposes, such as dividing devices into groups (by function, by vendor or by location), device commissioning or vendor-specific customer support cases before the device was commissioned (for example, identifying factory software version or related features).
```

Comment:
IMHO "vendor-specific customer support cases" may not tell much to a random passer-by. An alternative use-case to consider would be "finding applicable software updates". But that's just my feeling. Feel free to ignore :)